cdap-docs/_common/common-build.sh

```diff
@@ -291,16 +294,22 @@ function test_an_include() {
     new_md5_hash=`md5sum ${target} | awk '{print $1}'`
   fi
   
+  local result_message=""
```

Comment:
Problem is that this not really an array, but a series of messages, with different line spacing between them. They are grouped together, with extra lines between groups. If I switch to an array, I loose the ease of control over that, which to me is more important since the whole point of this is a log file that's easy to read.